lib/Factory/ModuleFactory.php

```diff
@@ -579,7 +579,7 @@ public function query($sortOrder = null, $filterBy = array())
         }
 
         if ($this->getSanitizer()->getInt('notPlayerSoftware', $filterBy) == 1) {
-            $body .= ' AND name <> "Player Software" ';
+            $body .= ' AND module <> "playersoftware" ';
```

Comment:
these need to be `'` and not `"` - MySQL does support `"` but it is not considered good practice to use them